File: src/main/java/com/juan/guess/GameFor.java
package com.juan.guess;

import java.util.Random;
import java.util.Scanner;

public class GameFor {
    public static void main(String[] args) {
        /*for (int i=1 ; i<=3 ; i++ ) {  //for迴圈
            System.out.println(i);
        }*/
        /*int sum = 0 ;
        for (int i=1 ; i<=100 ; i++){ //累加
            sum = sum + i ;
        }
        System.out.println(sum);
        for (int j=5 ; j>=1 ; j--) { //累減
            System.out.println(j);
        }*/
        int secret = new Random().nextInt(10) +1 ; //1~10
        System.out.println(secret);
        Scanner scanner = new Scanner(System.in);
        for (int i=1 ; i<=4 ; i++){
            System.out.print("Please enter a number("+i+"/4) :");
            int number = scanner.nextInt();
            System.out.println("第"+i+"次："+ number);
            if(number > secret) {
                System.out.println("Lower");
            } else if (number < secret){
                System.out.println("Higher");
            } else {
                System.out.println("Great, the number is " + secret);
                break;//i=5;
            }
        }
        System.out.println("END");
    }
}

File: src/main/kotlin/com/juan/GameFor.kt
package com.juan

import java.util.*

fun main() {
    var secret = Random().nextInt(10)+1
    println(secret)
    val scanner = Scanner(System.`in`)
    for (i in 1..5) { // 第一次i=1 , 第二次i=2 , ....第五次i=5
        print("Please enter a number: ")
        var number = scanner.nextInt()
        println("第${i}次 : $number") //i為特殊變數所以必須用{}
        //if(number == -1) break  //輸入為 -1 離開迴圈
        if(number < secret){
            println("higher")
        } else if (number > secret) {
            println("lower")
        } else {
            println("Great , the number is: $secret")
            break
        }
    }
}